presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -207,6 +207,20 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testInsertForDefaultColumn()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.default_table (a bigint, b bigint default 40)");
+        assertUpdate("INSERT INTO default_table (a) VALUES (1)", 1);
+        assertUpdate("INSERT INTO default_table VALUES (2, 3)", 1);
```

Comment:
test with insert `(4, null)`

Reply:
Have added it

Reply:
Please add tests for:
- column without default value (not inserted into)
- insert into other than first column
- other combination (`INSERT INTO default_table (b, a)...`)

Reply:
Please also add column `c` (last) which you insert to.
We need to make sure data is not being "shifted" from `c` to `b`.

Also, please make sure we have tests like that in `TestHiveIntegrationSmokeTest`, as Hive's handling of "extra" columns needs to be different than in any other existing connector.